pkg/authenticator/authenticator.go

```diff
@@ -312,3 +316,26 @@ func decodeFromPEM(PEMBlock []byte, publicCert *x509.Certificate, privateKey cry
 
 	return decodedPEM, nil
 }
+
+func consumeInjectClientCertError(path string) string {
+	// The log file will not exist in old Conjur versions
+	err := utils.VerifyFileExists(path)
+	if err != nil {
+		log.Warn(log.CAKC056, path)
+		return ""
```

Comment:
Wouldn't it be better to return `nil` instead?

Reply:
`string` has no `nil` value